acceptance/internal/materialized_config.go

```diff
@@ -0,0 +1,57 @@
+package internal
+
+import (
+	"encoding/json"
+	"time"
+)
+
+const MaterializedConfigFile = "out.config.json"
+
+type MaterializedConfig struct {
+	GOOS                 map[string]bool     `json:"goos,omitempty"`
+	CloudEnvs            map[string]bool     `json:"cloud_envs,omitempty"`
+	Local                *bool               `json:"local,omitempty"`
+	Cloud                *bool               `json:"cloud,omitempty"`
+	CloudSlow            *bool               `json:"cloud_slow,omitempty"`
+	RequiresUnityCatalog *bool               `json:"requires_unity_catalog,omitempty"`
+	RequiresCluster      *bool               `json:"requires_cluster,omitempty"`
+	RequiresWarehouse    *bool               `json:"requires_warehouse,omitempty"`
+	EnvMatrix            map[string][]string `json:"env_matrix,omitempty"`
+	Timeout              float64             `json:"timeout_seconds,omitempty"`
```

Comment:
Timeouts don't answer "where does this run?", do we really need them? You can always see them with -logconfig.

Reply:
I think it's good to see this too as it influences test execution but if you have concerns I'm also okay to remove it